src/components/Breadcrumbs/index.tsx

```diff
@@ -52,33 +52,34 @@ const Breadcrumbs: React.FC<IProps> = ({
       ? [
           {
             fullPath: "/",
-            text: t("page-index-meta-title"),
+            text: <Translation id="page-index-meta-title" />,
           },
         ]
       : []),
     ,
     ...sliced.map((path, idx) => {
       return {
         fullPath: slugChunk.slice(0, idx + 2).join("/") + "/",
-        text: t(path),
+        text: t(path), // TODO: fix i18n strings for path breadcrumbs
```

Comment:
I think most of the translations for breadcrumbs are in the `common` ns. Maybe we could import the `common` ns and use it here
```ts
const { t } = useTranslation(["common", "page-index"])

...
text: t(path, { ns: "common" })
```

Just an idea, we can tackle this in a separate PR.

Reply:
Okay... so turns out we do not have a string for `"bug-bounty": "Bug bounty",` in our `common.json` file yet... This is also a bug in production, but easy fix.. I'll add the string in English and this should get resolved in time with future Crowdin translations.

By adding this it works locally without adding "common" to the namespaces here

Reply:
Ooops, I take that back... need to add "common". Will leave it in